release-notes-v1.0.0.md

```diff
@@ -0,0 +1,110 @@
+v1.0.0
+
+### Components
+
+This release contains a reference implementation of [FIDO Device Onboard (FDO) Specification](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/).
+
+It includes 4 components:
+  * Protocol Reference Implementation (PRI): [pri-fidoiot](https://github.com/secure-device-onboard/pri-fidoiot) is a JAVA based implementation of all the components specified in the FDO Specification.
+  * Client SDK: [client-sdk-fidoiot](https://github.com/secure-device-onboard/client-sdk-fidoiot) is a C based implementation for the device component specified in the FDO Specification. Additionally, it supports an implementation of the device that uses TPM infrastructure.
+  * EPID Verification Service: [epid-verification-service](https://github.com/secure-device-onboard/epid-verification-service) is a wrapper service written on top of EPID SDK to assist FDO Rendezvous service and FDO Owner service to perform device signature verification for EPID based devices.
+  * Test: [test-fidoiot](https://github.com/secure-device-onboard/test-fidoiot) implements a test-suite that gets executed as part of continuous integration pipeline.
+
+### New Features
+
+pending
+
+### Supported Cryptographic Modes
+
+**pri-fidoiot** supports following cryptographic modes.
+
+* For device attestation, it supports Intel EPID keys based on StEPID10 and StEPID11 and ECC keys
```

Comment:
What does the St mean in "StEPID10" and "StEPID11" ?

Reply:
EPID 1.0 and EPID 1.1

Reply:
Fixed